components/Chat.tsx

```diff
@@ -0,0 +1,55 @@
+import { View, Text, StyleSheet, Image } from 'react-native';
+import React from 'react';
+
+type ChatProps = {
+  Name: string;
+  Image: string;
+  LastMessage: string;
```

Comment:
The conventions of this language are to make these pascalCase:
```suggestion
  name: string;
  image: string;
  lastMessage: string;
```